src/api/v4/filter/bounding-box-gmaps.js

```diff
@@ -53,4 +58,12 @@ BoundingBoxGoogleMaps.prototype.$getInternalModel = function () {
   return this._internalModel;
 };
 
+// Helper to check if an element is a leafletmap object
+function _isGoogleMap (element) {
+  if (!google) {
+    throw Error('Google maps is not defined');
```

Comment:
What about a CartoError?

Reply:
IMO this is a "runtime exception" not a cartoError

Reply:
I would use here the `_isGoogleMapsLoaded` function from the client.js file (may be moving it to a utils.js file)